model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -42,6 +43,5 @@ data class AnalyzerConfiguration(
     /**
      * Configuration of the SW360 package curation provider.
```

Comment:
Is removing the annotation from `Sw360StorageConfiguration` intended?

Reply:
Yes. It's superfluous if the whole class has the same annotation.